build/codeformat/formatimports.go

```diff
@@ -197,5 +197,9 @@ func FormatGoImports(file string, doChangedFiles, doWriteFile bool) error {
 		return err
 	}
 
+	if !doChangedFiles && !doWriteFile {
+		fmt.Printf("%s", formattedBytes)
```

Comment:
Here seems incorrect, at least, there should be a "\n" at the end. Otherwise the lines from different calls may be outputted in the same line.

And I am not sure do we really need it, it is is necessary, it needs more comments with a real case.

Reply:
For no particular reason, I just want it works like gofmt, see https://cs.opensource.google/go/go/+/master:src/cmd/gofmt/gofmt.go;l=300

Reply:
I would prefer to keep it as simple as possible if there is no real need.

There was a discussion in https://github.com/go-gitea/gitea/pull/18633#pullrequestreview-874045422 about showing the diff result when doing formatting, till now, there is still no such need.